src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -31,7 +31,14 @@
     Gamepad manipulator;
 
     Intake intake;
-    Climb climb;
+    Lift lift;
+
+    boolean liftRaiseButtonPressed = false;
+    boolean liftLowerButtonPressed = false;
+
+    // DEFAULT_PERIOD
+    static final double dt = DEFAULT_PERIOD; // seconds
```

Comment:
So there actually exists a `TimedRobot.getPeriod()` method. We should use that instead. So `dt` becomes a local variable in `teleopPeriodic` (with the other lift nudging code).